### PR TITLE
Added Menu class in menu.py and created check in taxi.py to initiate …

### PR DESCRIPTION
**Pull request by ivymarkwell**
…Menu()

### CODE REVIEW

**Review by PradhanSandip — CHANGES_REQUESTED** [review]
There are few syntax error  in taxi.py if you could fix it before i merge.

**Comment by ivymarkwell** [comment]
Please do not merge! This looks complete but is not. There is still a small bug when exiting the game. I would also like to do a lot of refactoring tomorrow night (3/21/18). Current state is functional, but still needs to be cleaned up.

**Review by PradhanSandip — APPROVED** [review]

**Pull request merged**
